src/js/shot-generator/Editor.js

```diff
@@ -129,6 +132,7 @@ const metersAsFeetAndInches = meters => {
 }
 
 const feetAndInchesAsString = (feet, inches) => `${feet}′${inches}″`
+const feetAndInchesAsString2nd = (feet, inches) => `${feet}'${inches}"`  //need these because sdf font doesn't have these glyphs
```

Comment:
Could call this `feetAndInchesAsStringASCII`.

Could we add the missing glyphs to the SDF?

Reply:
I tried adding them using Hiero, but got glitched glyphs, we could look into it some more